lab_1_keywords_tfidf/start.py

```diff
@@ -1,14 +1,15 @@
 """
 Frequency-driven keyword extraction starter
 """
-import json
 from pathlib import Path
+import json
```

Comment:
why did you change input order? 
by default, we import libraries alphabetically